tests/test_alchemy.py

```diff
@@ -138,7 +164,7 @@ def test_multicall(self):
             list(
                 obj.slug for obj in models.session.query(
                     models.MultiFieldModel.slug
-                )
+                ).order_by(models.MultiFieldModel.slug)
```

Comment:
Postgres is less forgiving on the order, and this assert would fail randomly because of the order.